From f3c2ecae6b0d2ab4bbe52dad8f7bd9b050b81bbd Mon Sep 17 00:00:00 2001
From: Timothy Gu <timothygu99@gmail.com>
Date: Thu, 15 Oct 2020 02:38:59 -0400
Subject: [PATCH 2/2] data: Remove TasksMax

fprintd can spawn new threads like the gmain thread, and the gusb
library can spawn more threads of its own.
---
 data/fprintd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/fprintd.service.in b/data/fprintd.service.in
index daca723..b61d36b 100644
--- a/data/fprintd.service.in
+++ b/data/fprintd.service.in
@@ -33,7 +33,6 @@ RestrictRealtime=true
 
 # Privilege escalation
 NoNewPrivileges=true
-TasksMax=1
 
 # Capabilities
 CapabilityBoundingSet=
-- 
2.28.0

